.github/workflows/issue-opened.yml

```diff
@@ -57,32 +57,37 @@ jobs:
 
             const electronVersion = select('heading:has(> text[value="Electron Version"]) + paragraph > text', tree)?.value.trim();
             if (electronVersion !== undefined) {
-              const major = semver.parse(electronVersion)?.major;
-              if (major) {
-                const versionLabel = `${major}-x-y`;
-                let labelExists = false;
-
-                try {
-                  await github.rest.issues.getLabel({
-                    owner,
-                    repo,
-                    name: versionLabel,
-                  });
-                  labelExists = true;
-                } catch {}
-
-                if (labelExists) {
-                  // Check if it's an unsupported major
-                  const { ElectronVersions } = await import('${{ github.workspace }}/node_modules/@electron/fiddle-core/dist/index.js');
-                  const versions = await ElectronVersions.create(undefined, { ignoreCache: true });
-
-                  const validVersions = [...versions.supportedMajors, ...versions.prereleaseMajors];
-                  if (!validVersions.includes(major)) {
-                    core.setOutput('unsupportedMajor', true);
-                    labels.push('blocked/need-info ❌');
+              // It's possible for multiple versions to be listed -
+              // for now check for comma or space separated version.
+              const versions = electronVersion.split(/, | /);
+              for (const version of versions) {
+                const major = semver.parse(version)?.major;
+                if (major) {
+                  const versionLabel = `${major}-x-y`;
+                  let labelExists = false;
+
+                  try {
+                    await github.rest.issues.getLabel({
+                      owner,
+                      repo,
+                      name: versionLabel,
+                    });
+                    labelExists = true;
+                  } catch {}
+
+                  if (labelExists) {
+                    // Check if it's an unsupported major
+                    const { ElectronVersions } = await import('${{ github.workspace }}/node_modules/@electron/fiddle-core/dist/index.js');
+                    const versions = await ElectronVersions.create(undefined, { ignoreCache: true });
+
+                    const validVersions = [...versions.supportedMajors, ...versions.prereleaseMajors];
+                    if (!validVersions.includes(major)) {
+                      core.setOutput('unsupportedMajor', true);
+                      labels.push('blocked/need-info ❌');
+                    }
+
+                    labels.push(versionLabel);
```

Comment:
```suggestion
                    labels.push(versionLabel);

                    // Prevent excessive tags / tag spam
                    if (labels.length >= 5) {
                      break;
                    }
```

Reply:
Our supported versions cap at 4, so I don't think we'd ever end up with too many!

Reply:
Ah, Great! 👍

Reply:
Looking at the code, all versions seem to be added as tags, no matter if supported or not.

Just if there's at least one unsupported version, the `blocked/need-info ❌` label will get added.

So I think a cap could still make sense.

Potential future improvement: Add the `blocked/need-info ❌` label only if no supported version is listed.

Reply:
Oh yeah i see what you mean now - missed that! Yes, I agree - i think we should add all labels that are supported (inherently capping at ~4), and `blocked/need-info ❌` if none are.

Reply:
Done in [58dd5f1](https://github.com/electron/electron/pull/44679/commits/58dd5f18961c373d5279ee6e4d2f3a801c5976f5)

Reply:
That works for me. Thanks!

Benefit: If someone opens an issue and adds an outdated version number, then updates it, the issue will no longer forever be stuck with an `17-x-y` version label.